### ISSUE TITLE
Object of class Symfony\Component\HttpFoundation\ServerBag could not be converted to string

### ISSUE THREAD

**Issue by sheriffmarley** [Laravel-Backpack/CRUD#1801]
# Bug report

### What I did:
I have a table called `servers`. It contains two columns `server_ip` and `server_name`. I can create them in the create view, but i'm not able to edit or delete it. If I try one of those I receive the error mentioned above.

### What I expected to happen:
Show the edit form or delete the entry

### What happened:
It throws an `ErrorExecption`
```
ErrorException (E_RECOVERABLE_ERROR)
Object of class Symfony\Component\HttpFoundation\ServerBag could not be converted to string
```
### What I've already tried to fix it:
Tried to rename the table. 
Tried a custom route in web.php where the deletion worked
```php
public function delete(Server $server)
    {
        $server->delete();
    }
```

### Backpack, Laravel, PHP, DB version:
Backpack: Latest,
Laravel: Latest,
PHP: 7.1.3
MySQL: 5.7.17

**Comment by sheriffmarley** [comment]
The json output if trying to delete an item:

```json
{
    "message": "Object of class Symfony\\Component\\HttpFoundation\\ServerBag could not be converted to string",
    "exception": "ErrorException",
    "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Support\\Str.php",
    "line": 338,
    "trace": [
        {
            "function": "handleError",
            "class": "Illuminate\\Foundation\\Bootstrap\\HandleExceptions",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Support\\Str.php",
            "line": 338,
            "function": "substr_replace"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Support\\Str.php",
            "line": 315,
            "function": "replaceFirst",
            "class": "Illuminate\\Support\\Str",
            "type": "::"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\QueryException.php",
            "line": 56,
            "function": "replaceArray",
            "class": "Illuminate\\Support\\Str",
            "type": "::"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\QueryException.php",
            "line": 39,
            "function": "formatMessage",
            "class": "Illuminate\\Database\\QueryException",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Connection.php",
            "line": 665,
            "function": "__construct",
            "class": "Illuminate\\Database\\QueryException",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Connection.php",
            "line": 624,
            "function": "runQueryCallback",
            "class": "Illuminate\\Database\\Connection",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Connection.php",
            "line": 333,
            "function": "run",
            "class": "Illuminate\\Database\\Connection",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Query\\Builder.php",
            "line": 2095,
            "function": "select",
            "class": "Illuminate\\Database\\Connection",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Query\\Builder.php",
            "line": 2083,
            "function": "runSelect",
            "class": "Illuminate\\Database\\Query\\Builder",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Query\\Builder.php",
            "line": 2569,
            "function": "Illuminate\\Database\\Query\\{closure}",
            "class": "Illuminate\\Database\\Query\\Builder",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Query\\Builder.php",
            "line": 2084,
            "function": "onceWithColumns",
            "class": "Illuminate\\Database\\Query\\Builder",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Eloquent\\Builder.php",
            "line": 516,
            "function": "get",
            "class": "Illuminate\\Database\\Query\\Builder",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Eloquent\\Builder.php",
            "line": 500,
            "function": "getModels",
            "class": "Illuminate\\Database\\Eloquent\\Builder",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Concerns\\BuildsQueries.php",
            "line": 77,
            "function": "get",
            "class": "Illuminate\\Database\\Eloquent\\Builder",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Eloquent\\Builder.php",
            "line": 326,
            "function": "first",
            "class": "Illuminate\\Database\\Eloquent\\Builder",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Eloquent\\Builder.php",
            "line": 356,
            "function": "find",
            "class": "Illuminate\\Database\\Eloquent\\Builder",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Support\\Traits\\ForwardsCalls.php",
            "line": 23,
            "function": "findOrFail",
            "class": "Illuminate\\Database\\Eloquent\\Builder",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Eloquent\\Model.php",
            "line": 1608,
            "function": "forwardCallTo",
            "class": "Illuminate\\Database\\Eloquent\\Model",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\backpack\\crud\\src\\PanelTraits\\Delete.php",
            "line": 26,
            "function": "__call",
            "class": "Illuminate\\Database\\Eloquent\\Model",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\backpack\\crud\\src\\app\\Http\\Controllers\\Operations\\DeleteOperation.php",
            "line": 22,
            "function": "delete",
            "class": "Backpack\\CRUD\\CrudPanel",
            "type": "->"
        },
        {
            "function": "destroy",
            "class": "Backpack\\CRUD\\app\\Http\\Controllers\\CrudController",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Controller.php",
            "line": 54,
            "function": "call_user_func_array"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\ControllerDispatcher.php",
            "line": 45,
            "function": "callAction",
            "class": "Illuminate\\Routing\\Controller",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Route.php",
            "line": 219,
            "function": "dispatch",
            "class": "Illuminate\\Routing\\ControllerDispatcher",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Route.php",
            "line": 176,
            "function": "runController",
            "class": "Illuminate\\Routing\\Route",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php",
            "line": 682,
            "function": "run",
            "class": "Illuminate\\Routing\\Route",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php",
            "line": 30,
            "function": "Illuminate\\Routing\\{closure}",
            "class": "Illuminate\\Routing\\Router",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\backpack\\crud\\src\\app\\Http\\Controllers\\CrudController.php",
            "line": 45,
            "function": "Illuminate\\Routing\\{closure}",
            "class": "Illuminate\\Routing\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 133,
            "function": "Backpack\\CRUD\\app\\Http\\Controllers\\{closure}",
            "class": "Backpack\\CRUD\\app\\Http\\Controllers\\CrudController",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php",
            "line": 53,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest.php",
            "line": 31,
            "function": "Illuminate\\Routing\\{closure}",
            "class": "Illuminate\\Routing\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 151,
            "function": "handle",
            "class": "Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php",
            "line": 53,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\app\\Http\\Middleware\\CheckIfAdmin.php",
            "line": 63,
            "function": "Illuminate\\Routing\\{closure}",
            "class": "Illuminate\\Routing\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 151,
            "function": "handle",
            "class": "App\\Http\\Middleware\\CheckIfAdmin",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php",
            "line": 53,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Middleware\\SubstituteBindings.php",
            "line": 41,
            "function": "Illuminate\\Routing\\{closure}",
            "class": "Illuminate\\Routing\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 151,
            "function": "handle",
            "class": "Illuminate\\Routing\\Middleware\\SubstituteBindings",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php",
            "line": 53,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\VerifyCsrfToken.php",
            "line": 75,
            "function": "Illuminate\\Routing\\{closure}",
            "class": "Illuminate\\Routing\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 151,
            "function": "handle",
            "class": "Illuminate\\Foundation\\Http\\Middleware\\VerifyCsrfToken",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php",
            "line": 53,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\View\\Middleware\\ShareErrorsFromSession.php",
            "line": 49,
            "function": "Illuminate\\Routing\\{closure}",
            "class": "Illuminate\\Routing\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 151,
            "function": "handle",
            "class": "Illuminate\\View\\Middleware\\ShareErrorsFromSession",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php",
            "line": 53,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Session\\Middleware\\StartSession.php",
            "line": 63,
            "function": "Illuminate\\Routing\\{closure}",
            "class": "Illuminate\\Routing\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 151,
            "function": "handle",
            "class": "Illuminate\\Session\\Middleware\\StartSession",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php",
            "line": 53,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Cookie\\Middleware\\AddQueuedCookiesToResponse.php",
            "line": 37,
            "function": "Illuminate\\Routing\\{closure}",
            "class": "Illuminate\\Routing\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 151,
            "function": "handle",
            "class": "Illuminate\\Cookie\\Middleware\\AddQueuedCookiesToResponse",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php",
            "line": 53,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Cookie\\Middleware\\EncryptCookies.php",
            "line": 66,
            "function": "Illuminate\\Routing\\{closure}",
            "class": "Illuminate\\Routing\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 151,
            "function": "handle",
            "class": "Illuminate\\Cookie\\Middleware\\EncryptCookies",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php",
            "line": 53,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 104,
            "function": "Illuminate\\Routing\\{closure}",
            "class": "Illuminate\\Routing\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php",
            "line": 684,
            "function": "then",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php",
            "line": 659,
            "function": "runRouteWithinStack",
            "class": "Illuminate\\Routing\\Router",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php",
            "line": 625,
            "function": "runRoute",
            "class": "Illuminate\\Routing\\Router",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php",
            "line": 614,
            "function": "dispatchToRoute",
            "class": "Illuminate\\Routing\\Router",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Kernel.php",
            "line": 176,
            "function": "dispatch",
            "class": "Illuminate\\Routing\\Router",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php",
            "line": 30,
            "function": "Illuminate\\Foundation\\Http\\{closure}",
            "class": "Illuminate\\Foundation\\Http\\Kernel",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\fideloper\\proxy\\src\\TrustProxies.php",
            "line": 57,
            "function": "Illuminate\\Routing\\{closure}",
            "class": "Illuminate\\Routing\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 151,
            "function": "handle",
            "class": "Fideloper\\Proxy\\TrustProxies",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php",
            "line": 53,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest.php",
            "line": 31,
            "function": "Illuminate\\Routing\\{closure}",
            "class": "Illuminate\\Routing\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 151,
            "function": "handle",
            "class": "Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php",
            "line": 53,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest.php",
            "line": 31,
            "function": "Illuminate\\Routing\\{closure}",
            "class": "Illuminate\\Routing\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 151,
            "function": "handle",
            "class": "Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php",
            "line": 53,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize.php",
            "line": 27,
            "function": "Illuminate\\Routing\\{closure}",
            "class": "Illuminate\\Routing\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 151,
            "function": "handle",
            "class": "Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php",
            "line": 53,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\CheckForMaintenanceMode.php",
            "line": 62,
            "function": "Illuminate\\Routing\\{closure}",
            "class": "Illuminate\\Routing\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 151,
            "function": "handle",
            "class": "Illuminate\\Foundation\\Http\\Middleware\\CheckForMaintenanceMode",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php",
            "line": 53,
            "function": "Illuminate\\Pipeline\\{closure}",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php",
            "line": 104,
            "function": "Illuminate\\Routing\\{closure}",
            "class": "Illuminate\\Routing\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Kernel.php",
            "line": 151,
            "function": "then",
            "class": "Illuminate\\Pipeline\\Pipeline",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Kernel.php",
            "line": 116,
            "function": "sendRequestThroughRouter",
            "class": "Illuminate\\Foundation\\Http\\Kernel",
            "type": "->"
        },
        {
            "file": "C:\\Program Files (x86)\\EasyPHP-Devserver-17\\eds-www\\laravel\\public\\index.php",
            "line": 55,
            "function": "handle",
            "class": "Illuminate\\Foundation\\Http\\Kernel",
            "type": "->"
        }
    ]
}
```

I looked at the trace myself and it seems that on 
```php
58 call_user_func_array
…\vendor\laravel\framework\src\Illuminate\Routing\Controller.php : 54 
```
the arguments are correct and 

then on the next entry there is that `serverbag` (Taken from the edit route)
```php
59 Backpack\CRUD\app\Http\Controllers\CrudController edit
…\vendor\laravel\framework\src\Illuminate\Routing\Controller.php : 54 
```

**Comment by sheriffmarley** [comment]
I found out that the issue is in the `getCurrentEntryId()` function at the return statement. In request is object called `server` which has a `ServerBag`. 

Is there a way to prevent this, or do I have to rename my `server` model to sth. else?

**Comment by sheriffmarley** [comment]
Edit: 
It seems that this line is causing the issue
```
$this->crud->setEntityNameStrings('server', 'servers');
```

changing the singular term to anything other solves the issue

**Comment by lucasdcrk** [comment]
Had the same issue, renaming the entity solved it.

**Comment by tabacitu** [comment]
Thank you for reporting this @sheriffmarley & @lucasdcrk . I can confirm the issue is happening - it was because at some point (in ```getCurrentEntryId()``` I discovered) the ```server``` name was conflicting with an already existing variable on the request.

I've corrected this on ```master```, and we should be able to use conflicting names too, including ```server```. It'll be available with a ```composer update``` later today, as ```3.6.6```, after I bundle it with a few more non-breaking changes and tag it.

Thanks again. Cheers!
